tests/test_ingest.py

```diff
@@ -123,20 +125,24 @@ def test_ingest_dense_matrix(self):
             '--genome-annotation',
             'Ensembl 94',
             '--matrix-file',
-            'gs://fake-bucket/tests/data/dense_matrix_19_genes_100k_cells.txt',
+            'gs://fake-bucket/tests/data/dense_matrix_19_genes_1000_cells.txt',
             '--matrix-file-type',
             'dense',
         ]
         ingest = self.setup_ingest(args)[0]
-        model = ingest.load_args[1]
-        # Ensure that 'ObjectID' in model is removed
-        # print(model)
+        models = ingest.load_args[0]
+        print(models)
+        for model in models:
+            # Ensure that 'ObjectID' in model is removed
+            del model['_id']
+            self.assertEqual(model, gene_models[model['name']])
+        # print(models)
 
         # Verify gene model looks as expected
-        mock_dir = 'dense_matrix_19_genes_100k_cells_txt'
-        expected_model = get_gene_model(mock_dir)
+        # mock_dir = 'dense_matrix_19_genes_100k_cells_txt'
```

Comment:
Removing commented-out code here (and elsewhere in this file) would help readability.